src/index.tsx

```diff
@@ -164,6 +187,13 @@ const Content: VFC<{ serverAPI: ServerAPI }> = ({}) => {
           <FaPlay color="deepskyblue" />, <FaPause color="deepskyblue" />. To
           reset, disable and re-enable <em>Pause on focus loss</em>.
           <br />
+          <strong>
+            <em>- Also on overlay</em>
+          </strong>
+          <br />
+          Pauses apps when interacting with Steam Overlay. Requires pause on
```

Comment:
`Requires pause on focus loss to be enabled.` Could potentially be removed since it will be functionally apparent when interacting with the options.

Maybe a slightly smaller font could be achieved with `font-size` styling (percent or em) so everything fits or a down arrow could be fixed with absolute positioning to make it clear to the user that there's is more text ?

Reply:
`style={{ fontSize: "80%" }}` on the main `<div>` of the help text looks good.

Reply:
![20230112104414_1](https://user-images.githubusercontent.com/1030423/212033208-4dfd8976-7853-45b5-bba4-80f9cbb30028.jpg)

Reply:
```ts
        <div style={{ fontSize: "80%" }}>
          <strong>
            <em>- Pause before Suspend</em>
          </strong>
          <br />
          Pauses all apps before system suspend. May fix audio issues.
          <br />
          <strong>
            <em>- Pause on focus loss</em>
          </strong>
          <br />
          Pauses apps not in focus automatically when switching between them.
          Changing the state of an app in this mode will sticky them{" "}
          <FaPlay color="deepskyblue" />, <FaPause color="deepskyblue" />. To
          reset, disable and re-enable <em>Pause on focus loss</em>.
          <br />
          <strong>
            <em>- Also on overlay</em>
          </strong>
          <br />
          Pauses apps when interacting with Steam Overlay. Requires pause on
          focus loss to be enabled.
          <br />
          <strong>
            <em>Applications will appear here.</em>
          </strong>
        </div>
```